conda_smithy/cli.py

```diff
@@ -88,6 +88,13 @@ def __call__(self, args):
                           "conda-recipe you want to build a feedstock for. Got {}".format(
                 args.recipe_directory))
 
+        # conda-build has some really fruity behaviour where it needs CONDA_NPY
+        # and CONDA_PY in order to even read a meta. Because we compute version
+        # matricies anyway the actual number makes absolutely no difference.
+        import conda_build.config
+        conda_build.metadata.config.CONDA_NPY = '99.9'
```

Comment:
Interesting. I guess it really can't. It's not like there are NumPy selectors. Has it done its jinja templating passes by this point?